packages/mobile/jest.setup.js

```
@@ -1,2 +1,170 @@
+      })),
+    },
+    from: jest.fn().mockReturnThis(),
+    select: jest.fn().mockReturnThis(),
+    insert: jest.fn().mockReturnThis(),
+    update: jest.fn().mockReturnThis(),
+    delete: jest.fn().mockReturnThis(),
+    eq: jest.fn().mockReturnThis(),
+    order: jest.fn().mockReturnThis(),
+    limit: jest.fn().mockReturnThis(),
+    single: jest.fn(() => Promise.resolve({ data: { id: 'test-id', title: 'Test Note' }, error: null })),
+    then: jest.fn(() => Promise.resolve({ data: [{ id: 'test-id', title: 'Test Note' }], error: null })),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)